packages/core/src/NoneTool.ts

```diff
@@ -0,0 +1,75 @@
+import path from 'path';
+import fs from 'fs-extra';
+
+import { Tool, ToolType, Package, PackageJSON, Packages, InvalidMonorepoError } from './Tool';
+import { expandPackageGlobs } from "./expandPackageGlobs";
+
+export const NoneTool : Tool = {
+    type: 'none',
+
+    async isMonorepoRoot(directory: string): Promise<boolean> {
+        try {
+            const pkgJson: PackageJSON = (await fs.readJson(path.join(directory, "package.json")));
+            return true;
+        } catch (err) {
+            if (err.code !== "ENOENT") {
+                throw err;
+            }
+        }
+        return false;
+    },
+
+    isMonorepoRootSync(directory: string): boolean {
+        try {
+            const pkgJson: PackageJSON = fs.readJsonSync(path.join(directory, "package.json"));
+            return true;
+        } catch (err) {
+            if (err.code !== "ENOENT") {
+                throw err;
+            }
+        }
+        return false;
+    },
+
+    async getPackages(directory: string): Promise<Packages> {
+        try {
+            const pkgJson: PackageJSON = (await fs.readJson(path.join(directory, "package.json")));
+            const pkg: Package = {
+                dir: directory,
+                packageJson: pkgJson
+            };
+
+            return {
```

Comment:
Related to [the other comment](https://github.com/Thinkmill/manypkg/pull/151#discussion_r1052147465)... `npmClient` would make sense here too 🤔 since a "single"/none project can freely use just any npm client and Changesets might have to use the correct one for publishing and stuff like that.

I'm not sure how this was wired up before and if we had any potential (unreported) issues with this.

Reply:
For the record - since this probably wasn't handled before, we can not worry about this here.